recipes/beyla/README.md

```diff
@@ -0,0 +1,141 @@
+# eBPF HTTP Observability with Beyla
+
+This recipe demonstrates how to configure the OpenTelemetry Collector
+(as deployed by the Operator) to send [Beyla](https://github.com/grafana/beyla)
+metric and trace data to GCP [Cloud Trace](https://cloud.google.com/trace) and
+[Google Managed Service for Prometheus](https://cloud.google.com/stackdriver/docs/managed-prometheus).
+
+In this recipe, Beyla is configured to collect http metrics and traces from all
+workloads in the cluster without any code changes. Beyla has other features,
+such as auto-instrumentation for Go applications, but this sample does not use
+it for that purpose.
+
+This recipe is based on applying a Collector config that enables the Google Cloud exporter.
+It provides an `OpenTelemetryCollector` object that, when created, instructs the Operator to
+create a new instance of the Collector with that config. If overwriting an existing `OpenTelemetryCollector`
+object (i.e., you already have a running Collector through the Operator such as the one from the
+[main README](../../README.md#starting-the-collector)), the Operator will update that existing
+Collector with the new config.
+
+
+## Prerequisites
+
+* Cloud Trace and Cloud Monitoring APIs enabled in your GCP project
+* The `roles/cloudtrace.agent` and `roles/monitoring.metricWriter`
+  [IAM permissions](https://cloud.google.com/trace/docs/iam#roles)
+  for your cluster's service account (or Workload Identity setup as shown below).
+* A running (non-autopilot) GKE cluster
```

Comment:
Consider highlighting the non-autopilot requirement and explaining _why_. e.g.,

> [!WARNING]  
> Beyla only works with GKE standard clusters, because GKE Auto restricts the use of blah blah feature.

Reply:
Addressed in https://github.com/GoogleCloudPlatform/opentelemetry-operator-sample/pull/51